### PR TITLE
New template structure

### PR DESCRIPTION
**Pull request by Clare2D**
Matching PR for the new template from create_interactive_report
- new user data
- remove user_id
- update file paths

depends on 2DegreesInvesting/create_interactive_report/pull/68

### CODE REVIEW

**Review by cjyetman — APPROVED** [review]
2DegreesInvesting/create_interactive_report/pull/68 is merged and I was able to run the web tools in the docker image successful if I used this PR in tandem, so I'm going to approve and merge this

**Comment by cjyetman** [comment]
can't merge until the merge conflicts are resolved though @Clare2D

**Comment by Clare2D** [comment]
Made changes. Will merge.

**Pull request merged**
